sfdx-project.json

```diff
@@ -0,0 +1,11 @@
+{
+    "packageDirectories": [
+        {
+            "path": "sfdx-source/apex-common",
+            "default": true
+        }
+    ],
+    "namespace": "",
+    "sfdcLoginUrl": "https://login.salesforce.com",
+    "sourceApiVersion": "48.0"
```

Comment:
I would consider leaving this at API 43 until you adopt the new SObjectDescribeOptions.DEFERRED option to avoid performance regressions in fflib_SObjectDescribe.

Reply:
@nawforce -- The API version here does not change the behavior of the apex classes.  This API version is how it engages with the deployment of code to the org.  The fflib_SObjectDescribe class api is controlled by the fflib_SObjectDescribe.cls-meta.xml file.